### DISCUSSION TITLE
Always have code-server open a workspace file

### DISCUSSION THREAD

**Discussion by ntimo** [coder/code-server#2154] [General] [answered]
Hello,
is it possible to always have code-server open a specific workspace file when you access it using a chrome web app? I know you can use a URL parameter but, I can't set that parameter in the chrome web app. And as far as I know there is now config file option for this.

**Comment by sr229** [comment, marked as answer]
if you execute your `code-server` instance to listen to a specific directory, it should automatically do it for you:

```
$ code-server ./folder
```

**Reply by code-asher** [reply]
:+1: Also works with workspace files.

**Reply by ntimo** [reply]
Yes but then I will have to edit the systemd file which to be honest I am not a big fan of. I think this should be a option of the config.yaml

**Reply by code-asher** [reply]
A configuration option sounds good to me.

**Reply by sr229** [reply]
A `working-directory` would be awesome actually.

**Comment by code-asher** [comment]
In addition to the comment above, code-server will remember your last opened directory or workspace so even if you don't specify it on the command line you should only have to open it once.

**Reply by ntimo** [reply]
Yes but that won't work if I switch over to another device like an iPad for example. Then I always have to open the workspace file manually. When I open Code server from the same browser that I had previously closed it remembers it correct.

**Reply by code-asher** [reply]
Hmm, that sounds like a bug. It should open the last workspace regardless of the browser or device used to access it.

**Reply by ntimo** [reply]
Should I create a bug ticket for that?

**Reply by sr229** [reply]
It's probable. It's not doing it's intended behavior.

**Reply by ntimo** [reply]
I just filed a bug report for this: https://github.com/cdr/code-server/issues/2161
